build-system/tasks/helpers.js

```diff
@@ -246,34 +245,19 @@ function compile(watch, shouldMinify) {
     });
   }
 
-  return Promise.all(promises)
-    .then(() => {
-      return compileJs('./src/', 'amp.js', './dist', {
-        toName: 'amp.js',
-        minifiedName: 'v0.js',
-        includePolyfills: true,
-        watch,
-        minify: shouldMinify,
-        wrapper: wrappers.mainBinary,
-        singlePassCompilation: argv.single_pass,
-        esmPassCompilation: argv.esm,
-      });
-    })
-    .then(() => {
-      if (!argv.single_pass) {
-        return compileJs('./src/', 'amp.js', './dist', {
-          toName: 'amp-esm.js',
-          minifiedName: 'v0-esm.js',
-          includePolyfills: true,
-          includeOnlyESMLevelPolyfills: true,
-          watch,
-          minify: shouldMinify,
-          wrapper: wrappers.mainBinary,
-        });
-      } else {
-        return Promise.resolve();
-      }
+  return Promise.all(promises).then(() => {
```

Comment:
Am confused, where is the secondary output for a `module` build?

Reply:
The is no secondary output. There will be a complete fresh run with `--esm` flag which will generate an ES2015 output to be copied in a different rtv folder.
@erwinmombay is working on the infrastructure PR to copy to a `/esm` folder inside our rtv folders

Reply:
Got it, makes sense.